README.md

```diff
@@ -1 +1,49 @@
-# java-jaxrs
\ No newline at end of file
+# OpenTracing JAX-RS Instrumentation
+
+Warning: This library is still a work in progress!
+
+[![Travis](https://travis-ci.org/opentracing-contrib/java-jaxrs.svg?branch=master)](https://travis-ci.org/opentracing-contrib/java-jaxrs)
+
+OpenTracing instrumentation for JAX-RS standard. 
+It supports server and client request tracing.
+
+## Tracing Server Requests
+```
+// register this in javax.ws.rs.core.Application
+ServerTracingDynamicFeature.Builder
+    .traceAll(yourPreferredTracer)
+    .withStandardTags()
```

Comment:
what's the purpose of this? In other words, when would someone NOT want standard tags? It sounds like an edge case, perhaps the standard tags should be the default, and if someone really wants to disable them they can call `withoutStandardTags()`